src/components/Profile/Components/TableComponent.tsx

```diff
@@ -0,0 +1,234 @@
+'use client'
+import * as utils from '@/utils'
+import * as interfaces from '@/interfaces'
+import Sorting from '@/components/Global/Sorting'
+import Loading from '@/components/Global/Loading'
+import { OptionsComponent } from './OptionsComponent'
+import * as consts from '@/constants'
+import { useCallback } from 'react'
+import { useRouter } from 'next/navigation'
+
+export const TableComponent = ({
+    data,
+    selectedTab,
+    currentPage,
+    itemsPerPage,
+}: {
+    data: interfaces.IProfileTableData[]
+    selectedTab: 'contacts' | 'history' | 'accounts' | undefined
+    currentPage: number
+    itemsPerPage: number
+}) => {
+    const router = useRouter()
+    const handleSendToAddress = useCallback(
```

Comment:
Same here

Reply:
I think If we want use Link component in Options component we need to change structure of Options component. I don't think it's worth the benefits we'll get. The structure is different than in the mobile view. What do you think? @borcherd